infra/scripts/smoke-tests/README.md

```
@@ -0,0 +1,178 @@
+- `SMOKE_CLIENT_GATEWAY_URL` - Client Gateway URL (required)
+
+**Retry Logic:**
+- Timeout: 10 seconds per request
+- Retries: 3 attempts with 5-second intervals
+
+**Exit Codes:**
+- `0` - all services are healthy
+- `1` - at least one service is unavailable
+
+**Output Example:**
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)